workflow/scripts/osemosys_global/transmission/main.py

```diff
@@ -53,57 +76,80 @@ def main(
+    gtd_exist_corrected, gtd_planned_corrected = correct_gtd_data(gtd_exist, 
+                                                                  gtd_planned, 
+                                                                  gtd_mapping, 
+                                                                  CUSTOM_TRN_BA_DICT_FROM, 
+                                                                  CUSTOM_TRN_BA_DICT_TO,
+                                                                  CUSTOM_TRN_BA_MISSING)
+    
+    # Set capital and fixed transmission costs.
+    cap_cost_trn, fix_cost_trn = get_transmission_costs(gtd_exist_corrected, 
+                                                                          gtd_planned_corrected,
+                                                                          cap_cost_base,
+                                                                          fix_cost_base,
+                                                                          centerpoints_dict, 
+                                                                          transmission_parameters, 
+                                                                          start_year, end_year, 
+                                                                          region_name, SUBSEA_LINES)
+    
+    # Calculate technology and transmission pathway specific transmission losses.
+    eff_trn = set_transmission_losses(gtd_exist_corrected, gtd_planned_corrected,
+                                      centerpoints_dict, transmission_parameters, SUBSEA_LINES)
+    
     # Set activity ratios for transmission.
     iar_trn, oar_trn = activity_transmission(iar_base, oar_base, 
-                                                               plexos_prop, interface_data,
-                                                               start_year, end_year,
-                                                               region_name)
-
-
+                                             eff_trn, start_year, 
+                                             end_year, region_name)
+    
     # Adjust activity limits if cross border trade is not allowed following user config.
-    df_crossborder_final = activity_transmission_limit(cross_border_trade, oar_trn)
+    activity_limit_trn = activity_transmission_limit(cross_border_trade, oar_trn)
     
-    # Set capital and fixed transmission costs.
-    df_cap_cost_trn_final, df_fix_cost_trn_final = get_transmission_costs(line_data, oar_trn,
-                                                                    cap_cost_base, fix_cost_base,
-                                                                    start_year, end_year,
-                                                                    region_name)
-
     # Set operational life for transmission.
-    df_op_life_trn_final = set_op_life_transmission(iar_trn, oar_trn, 
+    op_life_trn = set_op_life_transmission(iar_trn, oar_trn, 
```

Comment:
Does this function require both `iar_trn` and `oar_trn`? What transmission techs exist in one but not the other?

Reply:
No it doesn't! Fixed